docs/pass-through.md

```diff
@@ -0,0 +1,237 @@
+# Running Antrea In Pass-Through Mode
+
+Antrea supports chaining with other CNI implementations such as EKS, AKS. In this mode, 
```

Comment:
you use `AKS` / `EKS` in this document, while in the other one you used `aks` / `eks`. Could you make it uniform? Maybe use `AKS` / `EKS` everywhere.